Header header
string viewpoint
float32 ros_heading
